config/yaml_config.go

```diff
@@ -42,7 +42,7 @@ type YamlAgentConfig struct {
 		// A list of regex patterns that will exclude a process if matched.
 		BlacklistPatterns []string `yaml:"blacklist_patterns"`
 		// Enable/Disable the DataScrubber to obfuscate process args
-		ScrubArgs bool `yaml:"scrub_args"`
+		ScrubArgs *bool `yaml:"scrub_args,omitempty"`
```

Comment:
can you add a note on why this is a bool pointer as I did below? just for future readers :)